### DISCUSSION TITLE
Maven Snapshot vs. Release

### DISCUSSION THREAD

**Discussion by kjq** [renovatebot/renovate#28658] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

GitLab

### Please tell us more about your question or problem

When I have a Maven project (with the version as a <property>) and we set the value to `-SNAPSHOT` I want to keep it as a snapshot. 

So, essentially:
- if `X.y.z-SNAPSHOT` update to the latest snapshot
- if `X.y.z` update to the latest released version

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by rarkins** [comment]
Can you give some real examples with digits?

**Reply by kjq** [reply]
Give me a bit, I am now trying to figure out exactly what we are trying to do.  We use Maven Friendly CI Versions and it is causing all kinds of havoc with SNAPSHOTS vs. RELEASES.  Thanks though for the quick response.

**Reply by kjq** [reply, marked as answer]
The root of the problem is that in our BOM we introduced a "dummy" variable for the dependency versions because using Maven CI Friendly revisions + the Maven Flatten plugin was having issues.  When Renovate runs it is doing what it is supposed to and updating that variable but in this case we dont want _that_ variable to be updated.

Its not really an answer but it is the problem we are having.  Renovate is updating as it should but in this case for this multi-module project we do not want that specific variable to be updated.
